src/main.py

```diff
@@ -28,8 +30,9 @@
 def write_zarr(file_path: str, **dataset_kwargs):
     print(f'Writing {file_path}...')
     dataset = new_dataset(**dataset_kwargs)
-    # TODO (forman): zip here. so we don't need to do that manually
-    dataset.to_zarr(file_path)
+    dataset.to_zarr(file_path, consolidated=True)
```

Comment:
FWIW, the default is to write consolidated metadata, so this makes things more clear but is optional.

Reply:
Also, `to_zarr` can write a ZIP file directly, although I couldn't get it to work right.